flow/acac.cpp

```diff
@@ -1,19 +1,40 @@
+/*
+ * acac.cpp
+ *
+ * This source file is part of the FoundationDB open source project
+ *
+ * Copyright 2013-2024 Apple Inc. and the FoundationDB project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "flow/ActorContext.h"
+
 #ifdef WITH_ACAC
```

Comment:
Should this be the new macro (`ACTOR_MONITORING`)? Currently majority of the code is pre-processed out. If I change this code to use the new macro, then the code is not pre-processed out but then it also does not compile.

Reply:
They have to be rewritten to reflect the changes. I have only implemented `dumpActorCallBacktrace` but not the others. But it would be great if you can try `dumpActorCallBacktrace` in some ACTORs during the simulation tests.

Reply:
So... again, if you do not have time, I can make it later next month.

Reply:
> So... again, if you do not have time, I can make it later next month.
> 
> 

Ok, I can review this next month once you update it

Reply:
> > So... again, if you do not have time, I can make it later next month.
> 
> Ok, I can review this next month once you update it

This code is runnable though, if you got time can you put a `dumpActorCallBacktrace` to your code and see if it generates backtrace? At least it worked for me (I tested it on storage server/TLog)